packages/state-transition/test/perf/misc/rootEquals.test.ts

```diff
@@ -27,10 +27,10 @@ describe("root equals", () => {
   });
 
   bench({
-    id: "byteArrayEquals",
+    id: "Buffer.compare",
```

Comment:
this shouldn't be updated, it's the benchmark to compare those two against each other